manifests/map.pp

```diff
@@ -75,7 +75,8 @@
       owner   => 'root',
       group   => 'postfix',
       mode    => $mode,
-      require => [File["postfix map ${name}"], Exec["generate ${name}.db"]],
+      require => File["postfix map ${name}"],
+      notify  => $manage_notify,
```

Comment:
And I'm guessing the exec doesn't modify the rights if they are set before?

Reply:
that's correct.  if the file already exists, `postmap` doesn't change the existing permissions.